lib/mapper.js

```diff
@@ -8,6 +8,7 @@ var time = require('unix-time');
 var dot = require('obj-case');
 var flatten = require('flat');
 var is = require('is');
+var each = require('@ndhoule/each');
```

Comment:
It doesn't look like this is used anywhere. Shouldn't the style checks have caught this and errored?